adele-booking/src/main/java/com/microservicesteam/adele/model/data/BookingDo.java

```diff
@@ -0,0 +1,74 @@
+package com.microservicesteam.adele.model.data;
+
+import static java.util.stream.Collectors.toList;
+import static javax.persistence.CascadeType.PERSIST;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+import com.microservicesteam.adele.model.Booking;
+import com.microservicesteam.adele.model.PaymentStatus;
+
+@Entity
+public class BookingDo extends AbstractDo<Long> {
+
+    public final String paymentId;
+
+    public final PriceDo price;
```

Comment:
I noticed afterwards that `sumPrice` is not only the sum of ticket prices but multiply by discount as well.